xmtp_mls/src/groups/mod.rs

```diff
@@ -825,6 +877,50 @@ mod tests {
         messages.pop().unwrap()
     }
 
+    // Adds a member to the group without the usual validations on group membership
+    // Used for testing adversarial scenarios
+    async fn force_add_member<ApiClient: XmtpApi>(
```

Comment:
FYI, this helper is going to be useful for a lot more tests. It adds a member to the group without properly validating the commit, so you can create `GroupMembership` structs that are malicious and test if the receiving side properly rejects them.